src/main/java/seedu/address/model/person/Person.java

```diff
@@ -25,6 +25,16 @@
     /**
      * Every field must be present and not null.
      */
+
```

Comment:
Would it be possible for us to make the `Person` class abstract?

Reply:
Possible, just wondering if we want to keep a Person class!